Week-1/aryan29/api.py

```diff
@@ -0,0 +1,110 @@
+import requests
+import bs4
+import sys
+from termcolor import colored,cprint
+import json
+with open("user_config.json","r") as f:
+    conf=json.load(f)
+    if(len(conf["mob"])==0):
+        cprint("Please Configure your user_config.json file and add Code of Atleast 1 mobile Phone to generate json",'red')
+        exit(0)
```

Comment:
You should consider using `sys.exit` instead of `exit`.
(exit is a helper for the interactive shell - sys.exit is intended for use in programs)